app/views/state_file/questions/submission_confirmation/_penalty_interest_warning.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if @after_tax_deadline %>
+  <div class="notice--warning">
+    <p><%= t('state_file.questions.submission_confirmation.penalty_interest_warning.p1') %></p>
+    <p><%= t('state_file.questions.submission_confirmation.penalty_interest_warning.p2', billing_agency: t("state_file.questions.submission_confirmation.penalty_interest_warning.billing_agency.#{current_state_code}")) %></p>
```

Comment:
check for the presence of the string incase we add new states that don’t have it in the future